14-tidyr.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+layout: page
+title: R for reproducible scientific analysis
+subtitle: Dataframe manipulation with tidyr
+minutes: 60
+---
+
+```{r, include=FALSE}
+source("tools/chunk-options.R")
+opts_chunk$set(fig.path = "fig/12-plyr-")
+# Silently load in the data so the rest of the lesson works
+gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
+```
+
+> ## Learning objectives {.objectives}
+>
+> * To be able to convert dataframes between 'long' and 'wide' format with `tidyr`
```

Comment:
Do they also need to know what these mean? I would include that in the learning objectives as well. Many people may not be familiar with these concepts yet (unless they were specifically covered earlier).